app/AlexNet/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+if(UNIX)
+    unset(TENSORFLOW_FOUND)
+    find_path(tensorflow_INCLUDE_DIR
+            NAMES
+            tensorflow/core
+            tensorflow/c
+            HINTS
+            /usr/local/include/tensorflow
+            /usr/include/tensorflow)
+    find_library(tensorflow_LIBRARY NAMES tensorflow
+            HINTS
+            /usr/lib
+            /usr/local/lib)
+    include(FindPackageHandleStandardArgs)
+    find_package_handle_standard_args(tensorflow DEFAULT_MSG tensorflow_INCLUDE_DIR tensorflow_LIBRARY)
+    if(TENSORFLOW_FOUND)
+        set(TensorFlow_LIBRARIES ${tensorflow_LIBRARY})
+        set(TensorFlow_INCLUDE_DIRS ${tensorflow_INCLUDE_DIR})
+    else()
+        if (NOT DEFINED ENV{GITHUB_ACTIONS})
+          message(WARNING "TensorFlow not found. Skipping building of alexnet target.")
+          return()
```

Comment:
Please remove all of these ifs and make it universal

Reply:
the problem is that on the git and locally there are different ways and places to install the library and on the git it is definitely there because it is registered in ci.yml. And when we register a tensorflow search in cmakelist, then the check goes to github actions, so I divided these 2 processes

Reply:
I tried to define variables in the build step in ci.yml so that they would be visible there. I made a check for their emptiness there, so that the library would be build or not, but as a result, the build process fails due to checks in cmakelist

Reply:
do we need to do these checks with tensorflow now if we are going to work with another library we have built soon?